examples/api600/volume.rb

```diff
@@ -0,0 +1,200 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```

Comment:
Not sure why we have added a new file here. Can't we just reuse the existing example

Reply:
No we cannot use existing example as the request body for api300 and api500,600 are different. So there is a different example for API500 under examples/api500 folder and therefore I have created a separate file for api600 going by the directory name. We can come to a consensus as to how we can handle this as both api500 and api600 have similar examples and dont need 2 different files.

Reply:
yes. Thats what i meant. if there is no change between 500 and 600, then lets keep same example file.